lib/rules/declaration-property-value-whitelist/README.md

```diff
@@ -8,21 +8,31 @@ a { text-transform: uppercase; }
  * These properties and these values */
 ```
 
+This rule removes vendor prefixes before any checks.
+
 ## Options
 
-`object`: `{
-  "unprefixed-property-name": ["array", "of", "values"],
-  "unprefixed-property-name": ["/regex/", "non-regex"]
-}`
+Type: `object`  
+Structure:   
 
-If a property name is found in the object, only its whitelisted property values are allowed. This rule complains about all non-matching values. (If the property name is not included in the object, anything goes.)
+```json
+{
+  string: array|string
```

Comment:
Missing `regex` here.

Reply:
Add double quotes around the property and value to make this proper JSON